web/README.md

```diff
@@ -117,6 +117,14 @@ requests to the given target.
 > Keep in mind that you have to use a local user because social
 > logins (google/github) are not supported by development server.
 
+### WASM
+
+The web UI includes a WASM modul built from a Rust codebase located in `packages/teleport/src/ironrdp`.
```

Comment:
This PR does not include anything in `web/packages/teleport/src/ironrdp` and there is nothing there on master. I'm a little confused how this PR works without this source code. Does this README change belong in a different PR?

Reply:
This PR is being merged into the `ironrdp` branch, which is the big feature branch for the IronRDP client refactor:

https://github.com/gravitational/teleport/pull/26874